.mailmap

```diff
@@ -0,0 +1,108 @@
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <atarkowska@lifesci.dundee.ac.uk>
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <A.Tarkowska@dundee.ac.uk>
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <aleksandra-tarkowska@users.noreply.github.com>
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <atarkowska@05709c45-44f0-0310-885b-81a1db45b4a6>
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <Aleksandra Tarkowska@05709c45-44f0-0310-885b-81a1db45b4a6>
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <ola@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <Aleksandra Tarkowska@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <ola@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <ola@05709c45-44f0-0310-885b-81a1db45b4a6>
+Andreas Knab <andreas@glencoesoftware.com> <knabar@gmail.com>
+Andrea Falconi <andrea.falconi@gmail.com> <afalconi@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Andrea Falconi <andrea.falconi@gmail.com> <(no author)@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Andrew Patterson <ajpatterson@lifesci.dundee.ac.uk>
+Andrew Patterson <ajpatterson@lifesci.dundee.ac.uk> <workonly@qidane.com>
+Andrew Patterson <ajpatterson@lifesci.dundee.ac.uk> <andrew@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Andrew Patterson <ajpatterson@lifesci.dundee.ac.uk> <andrew@05709c45-44f0-0310-885b-81a1db45b4a6>
+Andrew Patterson <ajpatterson@lifesci.dundee.ac.uk> <ajpatterson@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Andrew Patterson <ajpatterson@lifesci.dundee.ac.uk> <ajpatterson@05709c45-44f0-0310-885b-81a1db45b4a6>
+Brian Loranger <bwzloranger@lifesci.dundee.ac.uk>
+Brian Loranger <bwzloranger@lifesci.dundee.ac.uk> <brian.loranger@lifesci.dundee.ac.uk>
+Brian Loranger <bwzloranger@lifesci.dundee.ac.uk> <bwzloranger@05709c45-44f0-0310-885b-81a1db45b4a6>
+Brian Loranger <bwzloranger@lifesci.dundee.ac.uk> <brain@05709c45-44f0-0310-885b-81a1db45b4a6>
+Brian Loranger <bwzloranger@lifesci.dundee.ac.uk> <TheBrain@05709c45-44f0-0310-885b-81a1db45b4a6>
+Carlos Neves <carlos@glencoesoftware.com> <cneves-x@05709c45-44f0-0310-885b-81a1db45b4a6>
+Carlos Neves <carlos@glencoesoftware.com> <carlos@05709c45-44f0-0310-885b-81a1db45b4a6>
+Carlos Neves <carlos@glencoesoftware.com> <staff@glencoesoftware.com>
+Chris Allan <callan@glencoesoftware.com> <callan@blackcat.ca>
+Chris Allan <callan@glencoesoftware.com> <cxallan@05709c45-44f0-0310-885b-81a1db45b4a6>
+Chris Allan <callan@glencoesoftware.com> <cxallan@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Chris Allan <callan@glencoesoftware.com> <cxallan@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Chris Allan <callan@glencoesoftware.com> <callan@05709c45-44f0-0310-885b-81a1db45b4a6>
+Chris Allan <callan@glencoesoftware.com> <callan@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Chris Allan <callan@glencoesoftware.com> <callan@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Carlos Neves <carlos@glencoesoftware.com>
```

Comment:
Is it expected that this one doesn't have a mapped address?

Reply:
Guess several don't. Those count as the "official" address?

Reply:
Yes, unlike the other lines which aim to replace different variants of the email by a canonical email address, these lines normalize commits using different variants of the author name